panther_lights/test/test_apa102.cpp

```diff
@@ -0,0 +1,97 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <panther_lights/test/mock_apa102.hpp>
+
+using MockAPA102 = panther_lights::mock_apa102::MockAPA102;
+
+class APA102Test : public ::testing::Test
```

Comment:
```suggestion
class TestAPA102 : public testing::Test
```
Just to be compatible with other tests